### ISSUE TITLE
path 'XXX' resolves to multiple networks

### ISSUE THREAD

**Issue by leonkyneur** [vmware/govmomi#266]
In my environment I have a Single Datacenter with multiple Clusters.

Each cluster may have an overlap in the Porgroup names.

For example my networking structure is such:
Datacenter1 -> Cluster1 -> dvSwitch0 -> pgApps
Datacenter1 -> Cluster2 -> Switch0 -> pgApps

Govmomi will display the following:
```
$ govc ls /Sydney/network 
/Datacenter1/network/pgApps
/Datacenter1/network/pgApps
```

When I attempt to use finder on these particular networks I get a MultipleFoundError.

Is there a way to structure the network so it's cluster aware? or should I uniquely name my portgroups across clusters?

**Comment by pietern** [comment]
The structure displayed by govc is the structure it gets back from vCenter. This means the port groups are part of the same root network folder for the datacenter.

The only way to fix this in govc itself is to allow specification of objects (networks, VMs, etc) by an attribute other than the name (or in addition to the name, where you would combine the name with an UUID or something). While possible, I'm not in favor of doing so, since this is where folders should really be used instead.

You can try to fix it with the current version of govc by moving the portgroups to different folders. Otherwise, the simplest way to solve this is to use different names.

**Comment by leonkyneur** [comment]
Thanks - sorting portgroups into folders will serve as a suitable solution for me.

**Comment by dougm** [comment]
Note that with https://github.com/vmware/govmomi/pull/2626 included in release 0.28, there are options to find networks by ID, rather than name. See: https://github.com/vmware/govmomi/blob/17e669d84193839acdbebe6aed5aea26b1c65d48/find/finder.go#L793-L803
